A-4E-C/Weapons/A4E_Weapons.lua

```diff
@@ -1964,9 +1964,9 @@ end
 local dispenser_data =
 {
     --use shapename,         bomblet,          bomblet_count
-    ["CBU-1/A"]          = { bomblet = BLU_4B_NEW_GROUP, bomblet_count = 19 },
```

Comment:
Sorry this is confusing but I set the bomblet count because each "bomblet" is actually a cluster group, the number of bomblets are encoded in this cluster group.